metron-interface/metron-alerts/src/app/alerts/configure-table/configure-table.component.ts

```diff
@@ -314,10 +316,15 @@ export class ConfigureTableComponent implements OnInit, AfterViewInit {
     });
   }
 
-  swapUp(index: number) {
+  swapUp(index: number, event: any) {
+    const colUpButtons = this.moveColUpBtn.toArray();
     if (index > 0) {
       [this.visibleColumns[index], this.visibleColumns[index - 1]] = [this.visibleColumns[index - 1], this.visibleColumns[index]];
     }
+    if (event.type === 'keyup') {
```

Comment:
The default behavior of the browser would cause the up arrow button to lose focus on enter keypress, which differs in behavior when compared to the down arrow button. This is because browsers treat enter keypress events on buttons as click events. 

I very rarely find justification in overriding default browser behavior, but I strongly feel that it would be very annoying for keyboard users to have to backtab 4 times for every spot they want to move a column up in display order.

Reply:
Well done. I see that we're the good guys here which is great but these are things that make it really difficult to understand two weeks later especially because we have to apply this approach only on the up button. 😅

Reply:
Point taken. I added a comment so we aren't looking back at this in 3 months and wondering what's going on here.

Reply:
Thank you @sardell, well done 👍